### DISCUSSION TITLE
Difference between Android app developer and software engineers?

### DISCUSSION THREAD

**Discussion by taotechs** [community/community#38474] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,
I'm find it challenging to differentiate between these two.
<b>Does it mean that an Android developer is also a software engineer?</b>
Like are they synonymous?
Can one start journey of Android Development from software engineering?
What's your suggestions
Thanks.

**Comment by david-engelmann** [comment, marked as answer]
I would say that Android app developer is a specific type of software engineer. Simply put, all Android developers are software engineers, but not all software engineers are Android developers. Generally speaking, the concepts you learn in software engineering can be applied to a plethora of sub topics, it's all a matter of what you want to work on!

**Reply by taotechs** [reply]
Thanks boss
As an Android developer to be,
Can I start with software engineering?

**Reply by david-engelmann** [reply]
Absolutely, I would start with the basics of programming! After getting acquainted, dive into building some simple apps!

**Reply by taotechs** [reply]
Thanks sir.
I really appreciate your answer

**Reply by david-engelmann** [reply]
It's a long and fun road! Welcome aboard!

**Reply by taotechs** [reply]
😃i believe i can do it.
I strongly do.
